src/odb/include/odb/db.h

```diff
@@ -7555,7 +7581,11 @@ class dbModBTerm : public dbObject
   void disconnect();
   void staSetPort(void* p);
   void* staPort();
-
+  bool isBusPort() const;
```

Comment:
Confusing - isBusMember is more precise

Reply:
The dbModBTerm can be one of three things (ulimately):
1. A regular modbterm port  (the default)
2. A bus port (in this case it has ordered members which are themselves modbterms)
3. A bundle port (in this case it has members in some collection which are themselves modbterms)

We are not in this api asking "am I a member of a bus port" we are saying "is this a bus port"

I think it a good idea to add the new api isBusMember too (could return a boolean and the index of the modbterm in the bus) but that is something else.

Reply:
also note the void* is now gone